server/datastore/sql/migrations/001-initial.sql

```diff
@@ -15,3 +15,21 @@ CREATE TABLE posts (
   postedAt INTEGER NOT NULL,
   FOREIGN KEY (userId) REFERENCES users (id)
 );
+
+CREATE Table comments (
+  id      VARCHAR PRIMARY KEY,
+  userId  VARCHAR NOT NULL,
+  PostId  VARCHAR NOT NULL,
+  Comment VARCHAR NOT NULL,
+  PostedAt INTEGER NOT NULL,
+  FOREIGN KEY (userId) REFERENCES users (id),
+  FOREIGN KEY (PostId) REFERENCES posts (id)
+)
+
+CREATE Table likes (
+  UserId  VARCHAR NOT NULL,
```

Comment:
same as above, camelCase instead of PascalCase.

Reply:
fixed